pkg/osgi_config.go

```diff
@@ -13,25 +14,35 @@ import (
 type OSGiConfig struct {
 	manager *OSGiConfigManager
 	pid     string
+	fpid    string
+	aemcId  string
```

Comment:
what it differ to fpid? fpid is generated? but aemcId is constant?

how about `fapid?` :) factory aemc pid